test/propagation.jl

```diff
@@ -108,12 +108,12 @@ using LinearAlgebra: norm
 
     etv = range(sol64.t0, sol64.t[end], 5)
```

Comment:
Now that `etv` represents days rather than seconds we could rename this to `tv`

Reply:
Done.